content/docs/glossary/transactions.mdx

```diff
@@ -67,7 +96,14 @@ To determine if a transaction is valid, many checks take place over the course o
   - The appropriate network passphrase was part of the transaction hash that was signed by each of the signers. See [Network Passphrases](./network-passphrase.mdx) for more.
   - The combined weight of all signatures for the source account _of the transaction_ meets the low threshold for the source account. This is necessary in order for fees to be taken and the sequence number to be incremented later in the transaction lifecycle.
 - **Memo** — The memo type must be a valid type, and the memo itself must be adhere to the formatting of the memo type.
-- **Time Bounds** — The transaction must be submitted within the set time bounds of the transaction, otherwise it will be considered invalid.
+- **Preconditions** — refer to the [Validity Conditions](#validity-conditions) described above for thorough explanations of each of these. For each of these that are set, the transaction will be considered invalid unless the transaction is submitted:
+  * **Time Bounds** — within the set time bounds of the transaction
+  * **Ledger Bounds** — within the set ledger bounds of the transaction
+  * **Minimum Sequence Number** — by a source account whose sequence number is larger than this value
+  * **Minimum Sequence Age** — by a source account who last changed its sequence number at least "x" long ago
```

Comment:
This is incorrect, it doesnt' have to have changed, but it is also hard to discuss it here, so I think we should just link to teh larger section.
```suggestion
  * **Minimum Sequence Age** — see [Minimum Sequence Age](#minimum-sequence-age).
```

Reply:
Yeah seems good, amended.